### PR TITLE
Style Radio component for In Loco

### PR DESCRIPTION
**Pull request by mairatma**
Also fixes styling for focused checkboxes.

Checkbox Storybook

### CODE REVIEW

File: src/site/modules/checkbox.overrides [hunk 1]
```diff
@@ -123,10 +188,18 @@
   .checkboxColor(@pink, @p300, @p100)
 }
 
+.ui.pink.checkbox.radio {
+  .radioColor(@pink, @p300, @p100, @p500)
```
Comment (TCezarRod) [inline src/site/modules/checkbox.overrides]: Idk if you noticed, but for the pink and green checkboxes (both radio and checkbox), the "active" background color are P50 and G50, respectively, not P100 and G100. The blue one does use B100, so I'm not sure if it was a mistake on the design, or if the difference was really intentional tho.

**Comment by TCezarRod** [comment]
The disabled unchecked radio has a subtle color change on `:hover`, I think that is not the expected behavior, or is it?
The same applies for the disabled not-radio checkbox, the hover animation is more visible on that one.

File: src/site/modules/checkbox.overrides (hunk 1 above)
Comment (mairatma) [reply src/site/modules/checkbox.overrides]: Hmm, truly weird, I'll ask them about this to make sure it's correct!

**Comment by mairatma** [comment]
@TCezarRod you have a very good eye for colors! 👏 
You're right, I hadn't noticed, I'll fix that!

File: src/site/modules/checkbox.overrides (hunk 1 above)
Comment (mairatma) [reply src/site/modules/checkbox.overrides]: @TCezarRod I've talked to Bruno and he said that this was on purpose, the light blue was too close to the gray used in the checkbox, so it needed to be darker :)

**Comment by mairatma** [comment]
I've fixed the hover problem, this pr can be reviewed again 😄

File: src/site/modules/checkbox.overrides (hunk 1 above)
Comment (TCezarRod) [reply src/site/modules/checkbox.overrides]: Fair enough.
So we really need the use @p50 and @g50 here, right?

**Comment by FranciscoGileno** [comment]
awesome!!!

**Comment by TCezarRod** [comment]
The hover on radio looks good to me now (: 
We can solve the problem with the regular checkbox in another PR, since it's kinda unrelated to this one. 

Everything else looks great :D

File: src/site/modules/checkbox.overrides (hunk 1 above)
Comment (mairatma) [reply src/site/modules/checkbox.overrides]: Oh gosh, you're right, I thought I had already done it like that 😂

Comment (mairatma) [reply src/site/modules/checkbox.overrides]: Done!

Comment (TCezarRod) [reply src/site/modules/checkbox.overrides]: 😄

**Comment by mairatma** [comment]
@TCezarRod Resolvi o problema do hover pra checkboxes também, era uma besteira, e dei uma checada geral nos dois (checkbox e radio). Valeu por notar essas coisas 😄

**Pull request merged**